### PR TITLE
Added Atom extensions and installation

### PR DESCRIPTION
**Pull request by JesusHerrera**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,68 @@
 
 ### Atom
 
-#### Extensions
+[Atom](https://atom.io/) is a highly customizable text editor. It's open source, developed by github and has tons of Packages to extend its functionality.
+
+#### Installation
+To install it just open a terminal run the following commands
```
Comment (SKOLZ) [inline README.md]: add osx installation and specify this one is for linux

File: README.md [hunk 2]
```diff
@@ -25,7 +25,68 @@
...
+##### Linter
+
+It shows errors in real time. Linter provides and Api so the other linters can display the error messages. You have to install a diferent linter for each language.
+
+![Image of Linter](https://i.github-camo.com/70b6e697c9d793642414b4ea6d08dbb9678877b3/687474703a2f2f672e7265636f726469742e636f2f313352666d6972507a322e676966)
+
+
+```
+apm install linter
+apm install linter-htmlhint
```
Comment (SKOLZ) [inline README.md]: whats this one for? 
we are going to add a pug linter, shouldn't we use that one instead?

File: README.md [hunk 3]
```diff
@@ -25,7 +25,68 @@
...
+apm install linter-csslint
+apm install linter-jshint
+apm install linter-scss-lint
```
Comment (SKOLZ) [inline README.md]: we have 2 scss linters, the one used in rails `scss-lint` and the one used in the FEB `sass-lint`. shouldn't we have one for sass-lint?. this packages detect your linter config file to show the errors right?

File: README.md (hunk 1 above)
Comment (JesusHerrera) [inline README.md]: I can't test the installation in OSX. I would like to someone else to do that.

Comment (JesusHerrera) [inline README.md]: @SKOLZ :point_up_2:

**Pull request merged**